src/entity/user/experiences/UserAwardExperience.ts

```diff
@@ -0,0 +1,30 @@
+import { ChildEntity, Column, JoinColumn, OneToOne } from 'typeorm';
+import { UserExperienceType } from './types';
+import { UserExperience } from './UserExperience';
```

Comment:
import `type`

Reply:
I'm actually extending this `UserExperience`, I cannot import as type